### PR TITLE
eZDFSFileHandlerMySQLiBackend: Added/fixed PHPDocs and shortened inline comments

### PR DESCRIPTION
**Pull request by jeromegamez**
Complete PHPDocs enhance the joy of coding :).

Cheers
:octocat: Jérôme

### CODE REVIEW

**Comment by jeromegamez** [comment]
Gentle ping @andrerom @bdunogier

File: kernel/private/classes/clusterfilehandlers/dfsbackends/mysqli.php [hunk 1]
```diff
@@ -1144,7 +1207,7 @@ protected function _selectOne( $query, $fname, $error = false, $debug = false, $
         $res = mysqli_query( $this->db, $query );
         if ( !$res )
         {
-            if ( mysqli_errno( $this->db ) == 1146 )
+            if ( mysqli_errno( $this->db )== 1146 )
```
Comment (andrerom) [inline kernel/private/classes/clusterfilehandlers/dfsbackends/mysqli.php]: cs

Comment (jeromegamez) [inline kernel/private/classes/clusterfilehandlers/dfsbackends/mysqli.php]: fixed

**Comment by andrerom** [comment]
Thanks @jeromegamez :)

**Pull request merged**